packages/core/src/2d/assembler/SlicedSpriteAssembler.ts

```diff
@@ -125,5 +122,32 @@ export class SlicedSpriteAssembler {
     renderer._bounds.transform(worldMatrix);
   }
 
-  static updateUVs(renderer: SpriteRenderer): void {}
+  static updateUVs(renderer: SpriteRenderer): void {
+    const { _chunk: chunk } = renderer;
+    const vertices = chunk._meshBuffer._vertices;
+    const spriteUVs = renderer.sprite._getUVs();
+    let index = chunk._vEntry.start + 3;
+    for (let i = 0; i < 4; i++) {
+      const rowU = spriteUVs[i].x;
+      for (let j = 0; j < 4; j++) {
+        vertices[index] = rowU;
+        vertices[index + 1] = spriteUVs[j].y;
+        index += 9;
+      }
+    }
+  }
+
+  static updateColor(renderer: SpriteRenderer): void {
+    const { _chunk: chunk } = renderer;
+    const { r, g, b, a } = renderer.color;
+    const vertices = chunk._meshBuffer._vertices;
+    let index = chunk._vEntry.start + 5;
+    for (let i = 0; i < 16; ++i) {
+      vertices[index] = r;
+      vertices[index + 1] = g;
+      vertices[index + 2] = b;
+      vertices[index + 3] = a;
+      index += 9;
+    }
+  }
```

Comment:
```ts
static updateColor(renderer: SpriteRenderer): void {
    const chunk = renderer._chunk;
    const vertices = chunk._meshBuffer._vertices;
    const color = renderer.color;
    let index = chunk._vEntry.start + 5;
    for (let i = 0; i < 16; ++i) {
      color.copyToArray(vertices, index);
      index += 9;
    }
  }
```

Reply:
Adding one more API call compared to direct invocation